src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,8 +89,10 @@ private void configureButtonBindings() {
     driverShootButton.and(operatorShootButton)
         .whileActiveOnce(new Shoot(Constants.SHOOT_SPEED, m_hood.getSetpoint(), m_shooter, m_hood, m_revolver), false);
 
-    climbButton.toggleWhenPressed(
-        new Climb(m_driveJoystick, RobotMap.JOYSTICK_AXIS.CLIMB_1, RobotMap.JOYSTICK_AXIS.CLIMB_2, m_climber));
+    climbButton.toggleWhenPressed(new Climb(
+        () -> (Math.pow(m_operatorJoystick.getRawAxis(RobotMap.JOYSTICK_AXIS.CLIMB_1), 2) + Math.pow(
+            m_operatorJoystick.getRawAxis(RobotMap.JOYSTICK_AXIS.CLIMB_2) * Constants.CLIMBER_MAX_PERCENT_OUTPUT, 2)),
```

Comment:
Let's update this to only use the right stick Y axis, rather than this root mean square thing.

Reply:
Cyr wants both axes for some reason

Reply:
Resolved in bbd7d1a